src/backoffice/shared/components/debug/debug.element.ts

```diff
@@ -0,0 +1,215 @@
+import { UUITextStyles } from '@umbraco-ui/uui-css/lib';
+import { css, html, nothing, TemplateResult } from 'lit';
+import { customElement, property, state } from 'lit/decorators.js';
+import { UmbContextDebugRequest } from '@umbraco-cms/context-api';
+import { UmbLitElement } from '@umbraco-cms/element';
+import { UmbModalService, UMB_MODAL_SERVICE_CONTEXT_TOKEN } from '@umbraco-cms/modal';
+
+@customElement('umb-debug')
+export class UmbDebug extends UmbLitElement {
+	static styles = [
+		UUITextStyles,
+		css`			
+			#container {
+				display: block;
+				font-family: monospace;
+
+				z-index: 10000;
+
+				position:relative;
+				width: 100%;
+				padding: 10px 0;
+			}
+
+			uui-badge {
+				cursor: pointer;
+			}
+
+			uui-icon {
+				font-size: 15px;
+			}
+
+			.events {
+				background-color: var(--uui-color-danger);
+				color: var(--uui-color-selected-contrast);
+				max-height: 0;
+				transition: max-height 0.25s ease-out;
+				overflow: hidden;
+			}
+
+			.events.open {
+				max-height: 500px;
+				overflow: auto;
+			}
+
+			.events > div {
+				padding: 10px;
+			}
+
+			h4 {
+				margin: 0;
+			}
+		`,
+	];
+
+	@property({ reflect: true, type: Boolean })
+	enabled = false;
+
+	@property({ reflect: true, type: Boolean })
+	useDialog = false;
```

Comment:
Let's call this "dialog" so it resembles the wording of the "enabled" property and looks better when using it

```suggestion
	dialog = false;
```